diff --git a/Makefile b/Makefile
index a537a68..ad49df0 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 CFLAGS  := -std=c99 -Wall -O2 -D_REENTRANT
-LIBS    := -lpthread -lm -lcrypto -lssl
+LIBS    := -lpthread -lm -lcrypto -lssl -lluajit
 
 TARGET  := $(shell uname -s | tr '[A-Z]' '[a-z]' 2>/dev/null || echo unknown)
 
@@ -10,9 +10,9 @@ else ifeq ($(TARGET), darwin)
 	# Per https://luajit.org/install.html: If MACOSX_DEPLOYMENT_TARGET
 	# is not set then it's forced to 10.4, which breaks compile on Mojave.
 	export MACOSX_DEPLOYMENT_TARGET = $(shell sw_vers -productVersion)
-	LDFLAGS += -pagezero_size 10000 -image_base 100000000
-	LIBS += -L/usr/local/opt/openssl/lib
-	CFLAGS += -I/usr/local/include -I/usr/local/opt/openssl/include
+	LDFLAGS += -ld_classic
+	LIBS += -L/opt/homebrew/opt/openssl/lib -L/opt/homebrew/opt/luajit/lib
+	CFLAGS += -I/usr/local/include -I/opt/homebrew/opt/openssl/include -I/opt/homebrew/opt/luajit/include/luajit-2.1
 else ifeq ($(TARGET), linux)
         CFLAGS  += -D_POSIX_C_SOURCE=200809L -D_BSD_SOURCE
 	LIBS    += -ldl
@@ -29,10 +29,10 @@ BIN  := wrk
 ODIR := obj
 OBJ  := $(patsubst %.c,$(ODIR)/%.o,$(SRC)) $(ODIR)/bytecode.o
 
-LDIR     = deps/luajit/src
-LIBS    := -lluajit $(LIBS)
-CFLAGS  += -I$(LDIR)
-LDFLAGS += -L$(LDIR)
+# LDIR     = deps/luajit/src
+#LIBS    := -lluajit $(LIBS)
+#CFLAGS  += -I$(LDIR)
+#LDFLAGS += -L$(LDIR)
 
 all: $(BIN)
 
@@ -44,22 +44,23 @@ $(BIN): $(OBJ)
 	@echo LINK $(BIN)
 	@$(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
 
-$(OBJ): config.h Makefile $(LDIR)/libluajit.a | $(ODIR)
+# $(OBJ): config.h Makefile $(LDIR)/libluajit.a | $(ODIR)
+$(OBJ): config.h Makefile | $(ODIR)
 
 $(ODIR):
 	@mkdir -p $@
 
 $(ODIR)/bytecode.o: src/wrk.lua
 	@echo LUAJIT $<
-	@$(SHELL) -c 'cd $(LDIR) && ./luajit -b $(CURDIR)/$< $(CURDIR)/$@'
+	@$(SHELL) -c 'cd $(LDIR) && luajit -b $(CURDIR)/$< $(CURDIR)/$@'
 
 $(ODIR)/%.o : %.c
 	@echo CC $<
 	@$(CC) $(CFLAGS) -c -o $@ $<
 
-$(LDIR)/libluajit.a:
-	@echo Building LuaJIT...
-	@$(MAKE) -C $(LDIR) BUILDMODE=static
+# $(LDIR)/libluajit.a:
+# 	@echo Building LuaJIT...
+# 	@$(MAKE) -C $(LDIR) BUILDMODE=static
 
 .PHONY: all clean
 .SUFFIXES:
diff --git a/src/hdr_histogram.c b/src/hdr_histogram.c
index 57c6324..52f57a5 100644
--- a/src/hdr_histogram.c
+++ b/src/hdr_histogram.c
@@ -7,7 +7,7 @@
 #include <stdlib.h>
 #include <stdbool.h>
 #include <math.h>
-#include <x86intrin.h>
+/* #include <x86intrin.h> */
 #include <assert.h>
 #include <stdio.h>
 #include <string.h>
